api-reference/50 Common/Object Structures/ExportPivotGridProps/topLeftCell.md

```diff
@@ -0,0 +1,25 @@
+---
+id: ExportPivotGridProps.topLeftCell
+type: CellAddress | String
+default: { row: 1, column: 1 }
+inheritsType: CellAddress
+---
+---
+##### shortDescription
+A cell used as a start position for export.
```

Comment:
`loadPanel.md` and `topLeftCell.md` can be inherited from the base object. Why do you need separate descriptions for PivotGrid?